src/make-http-request.coffee

```diff
@@ -1,4 +1,5 @@
-request = require 'superagent'
+cacheModuleConfig = { defaultExpiration: 10, verbose: true }
```

Comment:
Can we move these into https://github.com/zooniverse/panoptes-javascript-client/blob/master/lib/config.js and set `verbose` depending on the environment?